app/default/data.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+
+import requests
+from app.config import FLASK_ROOT
+from app.config import GET_APPLICATION_ENDPOINT
+
+
+def get_data(endpoint: str):
+    if endpoint[:8] == "https://":
```

Comment:
🤔 Try using [startswith](https://www.programiz.com/python-programming/methods/string/startswith) for better comprehension.

Reply:
oo i do love a new python method